content/applications/essentials/contacts.rst

```diff
@@ -0,0 +1,243 @@
+
+Name
+----
+
+First, fill in the name of the :guilabel:`Individual` or :guilabel:`Company`. This is how the name
+appears throughout the database. This field is **mandatory**.
+
+.. tip::
+   :guilabel:`Individual` contacts can have a :guilabel:`Company` contact linked to it. After
+   selecting :guilabel:`Individual`, a new :guilabel:`Company Name...` field appears below the
+   first name field.
+
+Address
+-------
+
+Next, enter the :guilabel:`Address` of the :guilabel:`Company` or :guilabel:`Individual`.
+
+.. tip::
+   If the :guilabel:`Individual` option is chosen, then the *type of address* can be chosen from a
+   drop-down menu. Options for this drop-down menu include: :guilabel:`Contact`, :guilabel:`Invoice
+   Address`, :guilabel:`Delivery Address`, :guilabel:`Other Address`, and :guilabel:`Private
+   Address`.
+
+Additional fields
+-----------------
+
+Additional details are included on the initial form. The following fields are available:
+
+- :guilabel:`VAT`: Value Added Tax number.
+- :guilabel:`Citizen Identification`: citizen or government identification number (only available
+  on :guilabel:`Individual`).
+- :guilabel:`Job Position`: list the job position of the :guilabel:`Individual` (only available on
+  :guilabel:`Individual`).
+- :guilabel:`Phone`: list phone number (with country code). Make a call, send an SMS, or WhatsApp
+  message by hovering over the field on the saved form, and clicking the desired option.
+- :guilabel:`Mobile`: list mobile phone number (with country code). Make a call, send an SMS, or
+  WhatsApp message by hovering over the field on the saved form, and clicking on the desired option.
+- :guilabel:`Email`: enter the email address with the domain.
+- :guilabel:`Website`: enter the full website address, starting with `http` or `https`.
+- :guilabel:`Title`: select :guilabel:`Doctor`, :guilabel:`Madam`, :guilabel:`Miss`,
+  :guilabel:`Mister`, :guilabel:`Professor`, or create a new one directly from this field.
+- :guilabel:`Tags`: enter preconfigured tags by typing them in the field, or clicking the drop-down
+  menu, and selecting one. To create a new one, type the new tag in the field, and click
+  :guilabel:`Create` from the resulting drop-down menu.
+
+Contacts & Addresses tab
+------------------------
+
+At the bottom of the contact form are several tabs. On the :guilabel:`Contacts & Addresses` tab,
+contacts can be added that are associated with a :guilabel:`Company` and related addresses. For
+example, a specific contact person for the company can be listed here.
+
+Multiple addresses can be added on both :guilabel:`Individual` and :guilabel:`Company` contacts. To
+do so, click :guilabel:`Add` in the :guilabel:`Contacts & Addresses` tab. Doing so reveals a
+:guilabel:`Create Contact` pop-up form, in which additional addresses can be configured.
+
+.. image:: contacts/contact-form-add-address.png
+   :align: center
+   :alt: Add a contact/address to the contact form.
```

Comment:
I think this is a better image to use in place of `contacts/contact_form.png` because it has the contact's details filled out, and addresses shown. 

I'd like to see this image used (retaken without the markup on the Add button) in place of `contacts/contact_form.png` :).

Then, I don't think it is necessary to use an image here in the Contacts & Addresses tab, just to show the Add button.
```suggestion
```